ui.apps/src/main/content/jcr_root/apps/venia/clientlibs/common/js/CommerceGraphqlApi.js

```diff
@@ -74,7 +76,12 @@
                     let skus = productData[item.name];
                     let media = variants.filter(v => skus.indexOf(v.product.sku) !== -1);
                     if (media && media.length > 0) {
-                        media.forEach( v => productsMedia[v.product.sku] = `${imageUrlPrefix}${v.product.media_gallery_entries[0].file}`);
+                        media.forEach(
+                            v =>
+                                (productsMedia[
```

Comment:
Same here, it's somehow sometimes adding silly line breaks.

Reply:
Rewrote the syntax a bit, to prevent the 120 character line break.